src/abi/json-abi-format.md

```diff
@@ -481,125 +359,116 @@ its JSON representation would look like:
 
 ```json
 {
-  "types": [
+  "concreteTypes": [
     {
-      "typeId": "2e38e77b22c314a449e91fafed92a43826ac6aa403ae6a8acb6cf58239fbaf5d",
-      "type": "()",
-      "components": [],
-      "typeParameters": null
+      "type": "([str[5]; 3], bool, b256)",
+      "concreteTypeId": "625531542be70834dd127e771101ac1014111718451bfae996d97abe700c66a5",
+      "metadataTypeId": 1,
+    },
+    {
+      "type": "[str[5]; 3]",
+      "concreteTypeId": "40c357685306e593eb4c4154377425853a7387ac5a6962d1d9198081a011d64a",
```

Comment:
In this example, this concrete type is not referenced anywhere; instead, its referenced metadata type is used [here](https://github.com/FuelLabs/fuel-specs/pull/599/files#diff-29c836429beedee654a72c37d1f883c1e5a0084d468e1446c61da13d7f88783dR407). Does this type have a role? If not, is there place for optimization here so that only the _used_ concrete types are included in the `concreteTypes` array?